### PR TITLE
fix(docs): update the client url to the alpha client

### PR DESCRIPTION
**Pull request by iancoffey**
I had to use the alpha client to get things working, seems like this should be updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ $ kubectl get service deis-workflow  # note the IP address for later
 Then open an SSH session to a Kubernetes minion node, install the `deis` client, and register:
 
 ```console
-$ curl -sSL http://deis.io/deis-cli/install.sh | sh
+$ curl -sSL http://deis.io/deis-cli/install-v2-alpha.sh | sh
```
Comment (bacongobbler) [inline README.md]: Let's change the pipe to `bash` so it'll work on Linux. I'd link to the issue on deis/workflow but I'm on mobile :)

**Comment by helgi** [comment]
deis/workflow#153

**Comment by bacongobbler** [comment]
Wingman @helgi to the rescue :+1:

**Comment by arschles** [comment]
looks like a dupe. should this be closed?

**Comment by bacongobbler** [comment]
@arschles where is the duplicate for this?

**Comment by arschles** [comment]
isn't the duplicate change at https://github.com/deis/workflow/pull/153/files#diff-04c6e90faac2675aa89e2176d2eec7d8R45?

**Comment by bacongobbler** [comment]
two separate projects. This changes deis/chart's README where that PR changed deis/workflow's README.

**Comment by arschles** [comment]
sorry, missed that fact. ignore

**Pull request merged**